Packs/ApiModules/Scripts/JSONFeedApiModule/JSONFeedApiModule.py

```diff
@@ -125,7 +129,12 @@ def fetch_indicators_command(client: Client, indicator_type: str, feedTags: list
                 if not current_indicator_type:
                     continue
 
-                indicator = {'value': indicator_value, 'type': current_indicator_type, 'fields': {'tags': feedTags}}
+                indicator = {'value': indicator_value, 'type': current_indicator_type,
+                             'fields': {
+                                 'tags': feedTags,
+                                 'trafficlightprotocol': client.tlp_color
+                             }
+                             }
```

Comment:
```suggestion
                indicator = {
                	'value': indicator_value,
                	'type': current_indicator_type,
                    'fields': {
                    	'tags': feedTags,
                    	'trafficlightprotocol': client.tlp_color
                    }
                }
```
You have strange formatting here.